english/scream/scream.md

```diff
@@ -1,11 +1,15 @@
 Now you have screamed and it turns out to be that this is reality and you have
 to deal with the situation.
 
+But, first, you need to remove the marshmallow from your mouth.
+
 You will do:
 
 [Jump out of joy and eat?](eat/jumpandeat.md)
 
 [Call your mom?](mom/eatwithmom.md)
 
-[Its right time to go for walk](walk/walkwiththought.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.